src/hooks/useConnectionStatus.ts

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from "react"
+import NetInfo from "@react-native-community/netinfo"
+
+export const useConnectionStatus = (): boolean => {
+  const [isConnected, setIsConnected] = useState<boolean>(true)
```

Comment:
How do you feel about keeping the naming from the `NetInfo` library since we are doing a wrapper? Isn't `isInternetReachable` a little more descriptive and specific than `isConnected`?